### PR TITLE
OSDOCS#6399: OCP on ARM 4.14 release notes

### PR DESCRIPTION
**Pull request by kelbrown20**
**Version:** 4.14
**Issue:** [OSDOCS-6399](https://issues.redhat.com//browse/OSDOCS-6399)

**Link to docs preview:**

[Release notes -> OpenShift Container Platform on 64-bit ARM ](https://65657--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-installation-and-update)

**QE review:**
- [x] QE approval

### CODE REVIEW

**Comment by kelbrown20** [comment]
@aleskandro Hey Alessandro! Can you PTAL at this PR for ARM when you get the chance? Thank you!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -132,10 +132,14 @@ This installation method is an alternative to uploading an {op-system} image on
 Before you set a path value for the `template` parameter, ensure that the default {op-system} boot image in the {product-title} release matches the {op-system} image template or virtual machine version; otherwise, cluster installation might fail.
 ====
 
+[id="ocp-4-14-OCP-on-ARM"]
+==== {product-title} on 64-bit ARM
+
+{product-title} {product-version} is now supported on 64-bit ARM architecture-based Google Cloud Platform installer-provisioned and user-provisioned infrastructures. You can also now use the `oc mirror` CLI plug in disconnected environments on 64-bit ARM clusters. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.adoc#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:138]:
```suggestion
{product-title} {product-version} is now supported on 64-bit ARM architecture-based Google Cloud Platform installer-provisioned and user-provisioned infrastructures. You can also now use the `oc mirror` CLI plug-in disconnected environments on 64-bit ARM clusters. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.adoc#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p

Comment (kelbrown20) [reply release_notes/ocp-4-14-release-notes.adoc:138]: Nice catch! Thank you!

**Pull request merged**